src/AuthCallback.js

```diff
@@ -0,0 +1,28 @@
+// Copyright 2023 The Casdoor Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {useEffect} from "react";
+import * as Setting from "./Setting";
+import {setToken} from "./Setting";
+
+export const AuthCallback = () => {
+  useEffect(() => {
+    Setting.CasdoorSDK.exchangeForAccessToken().then((res) => {
```

Comment:
Why is client secret not needed here?

Reply:
Pure front application put secret is unsafe,  the function useing pkce process in casdoor-js-sdk , https://datatracker.ietf.org/doc/html/rfc7636 is dedicated to the scene. So here don't need a secrect.